packages/valory/connections/test_libp2p/tests/base.py

```diff
@@ -453,3 +436,65 @@ def client_multiplexers(self) -> List[Multiplexer]:
     def mailbox_multiplexers(self) -> List[Multiplexer]:
         """Mailbox multiplexers"""
         return self._multiplexers_by_connection_type(P2PLibp2pMailboxConnection)
+
+
+class BaseP2PLibp2pAEATestCaseMany(AEATestCaseMany):
+    """BaseP2PLibp2pAEATestCaseMany"""
```

Comment:
factored this out since I will reuse it in the robustness tests in a subsequent PR